src/main.ts

```diff
@@ -1,13 +1,14 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app/app.module';
-import { config } from '@app/common';
 import { ValidationPipe } from '@nestjs/common';
 
+import * as passport from 'passport';
+import session = require('express-session');
+
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-
   app.enableCors({
-    origin: config.FRONTEND_URL,
+    origin: '*',
```

Comment:
This should be config.FRONTEND_URL not '*'

Also use the config service here if possible instead.

Reply:
done, this was only for postman check